### ISSUE TITLE
"Already have an account?" link not working

### ISSUE THREAD

**Issue by yasoob** [team-alembic/ash_authentication_phoenix#295]
Hi guys! I followed the [Ash auth with Phoenix](https://hexdocs.pm/ash_authentication_phoenix/getting-started-with-ash-authentication-phoenix.html) guide completely and got a basic form up and going in no time. However, there was a slight hiccup. The "Already have an account?" link isn't working. It sends an event to the server:

```
[debug] HANDLE PARAMS in AshAuthentication.Phoenix.SignInLive
  Parameters: %{}
[debug] Replied in 19µs
```

But nothing happens. The UI doesn't change.

Here is a small video showing the error:

https://github.com/team-alembic/ash_authentication_phoenix/assets/3696393/3932b6cc-a8e5-471f-ba92-f899a739d942

**Comment by zachdaniel** [comment]
🤔 any console errors or any errors in the logs?

**Comment by yasoob** [comment]
Nope. I already pasted what came up in terminal logs. There is nothing in the browser console.

**Comment by zachdaniel** [comment]
Can I see your router where you've added the ash_authentication routes? Are you on the latest version of both `ash_authentication` and `ash_authentication_phoenix`?

**Comment by yasoob** [comment]
My complete router file:

```
defmodule DaakiaWeb.Router do
  use DaakiaWeb, :router
  use AshAuthentication.Phoenix.Router

  pipeline :browser do
    plug :accepts, ["html"]
    plug :fetch_session
    plug :fetch_live_flash
    plug :put_root_layout, html: {DaakiaWeb.Layouts, :root}
    plug :protect_from_forgery
    plug :put_secure_browser_headers
    plug :load_from_session
  end

  pipeline :api do
    plug :accepts, ["json"]
    # Ash
    plug :load_from_bearer
  end

  scope "/", DaakiaWeb do
    pipe_through :browser

    get "/", PageController, :home

    # Ash auth
    sign_in_route()
    sign_out_route AuthController
    auth_routes_for Daakia.Accounts.User, to: AuthController
    reset_route []
  end

  # Other scopes may use custom stacks.
  # scope "/api", DaakiaWeb do
  #   pipe_through :api
  # end

  # Enable LiveDashboard and Swoosh mailbox preview in development
  if Application.compile_env(:daakia, :dev_routes) do
    # If you want to use the LiveDashboard in production, you should put
    # it behind authentication and allow only admins to access it.
    # If your application does not have an admins-only section yet,
    # you can use Plug.BasicAuth to set up some basic authentication
    # as long as you are also using SSL (which you should anyway).
    import Phoenix.LiveDashboard.Router

    scope "/dev" do
      pipe_through :browser

      live_dashboard "/dashboard", metrics: DaakiaWeb.Telemetry
      forward "/mailbox", Plug.Swoosh.MailboxPreview
    end
  end
end
```

My mix file:

```
      {:ash, "~> 2.15.8"},
      {:ash_authentication, "~> 3.11.15"},
      {:ash_authentication_phoenix, "~> 1.8.4"},
      {:ash_postgres, "~> 1.3.52"}
```

**Comment by zachdaniel** [comment]
Hmmmmmm....it's working for me 😆 I'm not sure what the issue could be, perhaps James will know when he's back around next week? In the meantime, https://ash-hq.org is open source here: https://github.com/ash-project/ash_hq/tree/main and has a mostly conventional `ash_authentication` setup to use as a reference, perhaps you can spot the difference?

**Comment by zachdaniel** [comment]
This guide should help on that front: https://alembic.com.au/blog/customising-ash-authentication-with-phoenix-liveview

**Comment by yasoob** [comment]
Ok so I just checked the page source. On ash hq page, this is what the "Already have an account?" link looks like:

<img width="1210" alt="image" src="https://github.com/team-alembic/ash_authentication_phoenix/assets/3696393/af95b686-d975-474f-be5e-ce288c5ec1c0">

However, on my new website, this is what it looks like:

<img width="1211" alt="image" src="https://github.com/team-alembic/ash_authentication_phoenix/assets/3696393/28b65e93-a3fe-4425-8231-7d3851e3c5e3">

Does this give you any clue on what might be wrong? Is it something I did that might have caused this or was there a change in the ash_auth codebase?

**Comment by zachdaniel** [comment]
okay, interesting, I think this may be a regular old-fashioned bug :)

**Comment by yasoob** [comment]
I spent the last couple minutes trying to pinpoint the bug location. Narrowed it down to [this part](https://github.com/team-alembic/ash_authentication_phoenix/blob/main/lib/ash_authentication_phoenix/components/password.ex#L294-L306) and then I checked this issue again and you already have a PR out. Love it! 

I was initially under the impression that there is no way to directly link to the register and reset password pages. But upon closer reading of the code, it seems like we can specify custom URLs. Sweet!
